en/docs/learn/analytics/configuring-alerts.md

```diff
@@ -68,14 +68,15 @@ The users of your APIs can subscribe to analytics-related alerts from the API Pu
                 name: email
                 namespace: sink
                 properties:
-                username: <senders-email-username>
-                address: <senders-email-address>
-                password: <senders-email-password>
-                mail.smtp.host: <senders-email-host>
-                port: 25
-                ssl.enable : false
-                auth : false
-                mail.store.protocol: pop3
+                    username: <senders-email-username>
+                    address: <senders-email-address>
+                    password: <senders-email-password>
+                    host: <senders-email-host>
+                    port: 25
+                    ssl.enable : false
+                    auth : false
+                    mail.store.protocol: pop3
+                    mail.smtp.starttls.enable: true
```

Comment:
Can you maintain the original indentation and update only the required line?

Reply:
with original indentation there was an error in logs, indentation is definitely required after this configuration only it started working
I will appreciate if could merger

Reply:
any update?

Reply:
any update, please?